tests/fis/test_fis_probes.py

```diff
@@ -0,0 +1,58 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from chaoslib.exceptions import FailedActivity
+
+from chaosaws.fis.probes import get_experiment
+
+
+def test_that_fis_action_modules___all___attribute_exposed_correctly():
```

Comment:
Would this be a more appropriate function name:
`test_that_fis_probe_modules___all___attribute_exposed_correctly()`

Reply:
Bother. What I get for using my other code as references.